MC/analysis_testing/o2dpg_analysis_test_workflow.py

```diff
@@ -128,20 +128,25 @@
                      "cmd": "o2-analysis-timestamp {CONFIG} | o2-analysis-track-propagation {CONFIG} | o2-analysis-trackextension {CONFIG} | o2-analysis-trackselection {CONFIG} | o2-analysis-event-selection {CONFIG} | o2-analysis-multiplicity-table {CONFIG} | o2-analysis-mm-dndeta-mft {CONFIG} {AOD}"}
 ANALYSES.append(analysis_PWGMMMFT)
 analysis_EventSelectionQA = {"name": "EventSelectionQA",
-                         "expected_output": ["AnalysisResults.root"],
-                         "valid_for": [ANALYSIS_VALID_MC, ANALYSIS_VALID_DATA],
-                         "cmd": "o2-analysis-timestamp {CONFIG} | o2-analysis-track-propagation {CONFIG} | o2-analysis-event-selection {CONFIG} | o2-analysis-event-selection-qa {CONFIG} {AOD}"}
+                             "expected_output": ["AnalysisResults.root"],
+                             "valid_for": [ANALYSIS_VALID_MC, ANALYSIS_VALID_DATA],
+                             "cmd": "o2-analysis-timestamp {CONFIG} | o2-analysis-track-propagation {CONFIG} | o2-analysis-event-selection {CONFIG} | o2-analysis-event-selection-qa {CONFIG} {AOD}"}
 ANALYSES.append(analysis_EventSelectionQA)
 analysis_WeakDecayTutorial = {"name": "WeakDecayTutorial",
                               "expected_output": None,
                               "valid_for": [ANALYSIS_VALID_MC],
                               "cmd": "o2-analysis-track-propagation {CONFIG} | o2-analysistutorial-weak-decay-iteration {CONFIG} {AOD}"}
 ANALYSES.append(analysis_WeakDecayTutorial)
-analysis_WeakDecayTutorial = {"name": "CheckDataModelMC",
-                              "expected_output": ["AnalysisResults.root"],
-                              "valid_for": [ANALYSIS_VALID_MC],
-                              "cmd": "o2-analysis-track-propagation {CONFIG} | o2-analysis-check-data-model-mc {CONFIG} {AOD}"}
-ANALYSES.append(analysis_WeakDecayTutorial)
+analysis_CheckDataModelMC = {"name": "CheckDataModelMC",
+                             "expected_output": ["AnalysisResults.root"],
+                             "valid_for": [ANALYSIS_VALID_MC],
+                             "cmd": "o2-analysis-track-propagation {CONFIG} | o2-analysis-check-data-model-mc {CONFIG} {AOD}"}
```

Comment:
This one shouldn't neet the track propagation

Reply:
Thanks, I have removed the dependency